pkg/app/piped/cmd/piped/piped.go

```diff
@@ -624,14 +626,27 @@ func (p *piped) sendPipedMeta(ctx context.Context, client pipedservice.Client, c
 		})
 	}
 
+	copiedCfg := &config.PipedSpec{}
+	err := deepcopy(copiedCfg, cfg)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: just to follow our convention
```suggestion
	if err := deepcopy(copiedCfg, cfg); err != nil {
		return err
	}
```

Reply:
I fixed it :bow: